spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployer.java

```diff
@@ -545,8 +545,8 @@ public Release upgradeStream(String streamName, PackageIdentifier packageIdentif
 	 * @param streamName the name of the stream to rollback
 	 * @param releaseVersion the version of the stream to rollback to
 	 */
-	public void rollbackStream(String streamName, int releaseVersion) {
-		this.skipperClient.rollback(streamName, releaseVersion);
+	public Release rollbackStream(String streamName, int releaseVersion) {
```

Comment:
This isn't an API change and I believe this is OK to do it for the point release (given this is only used inside the service API - at the SkipperStreamDeployer)